packages/common/src/core/HubItemEntity.ts

```diff
@@ -161,6 +161,14 @@ export abstract class HubItemEntity<T extends IHubItemEntity>
     return this.entity.canDelete;
   }
 
+  /**
+   * The orgId of the Entity, if available
+   * @returns
+   */
+  get orgId() {
```

Comment:
We have a requirement that the `/discussions/:id` routes can only be accessed if the item is associated with the site. Adding `orgId` here so we can leverage the [item-in-site](https://github.com/ArcGIS/opendata-ui/blob/master/packages/ember-arcgis-opendata-components/addon/utils/item-in-site.js#L1) utility from opendata-ui. This begs the question if that should maybe be a custom permission that can be applied to entities, but felt that was outside the scope of this story.

Reply:
I agree this feels like something that should be handled by the permissions system, but Dave would know better, and I think this is a fine solution for now

Reply:
I overlooked that the `HubSite` entity has a [contains](https://github.com/Esri/hub.js/blob/f/5906-discussions-entity/packages/common/src/sites/HubSite.ts#L246) method, which checks to see if an entity is in the site's catalog, but that only satisfies part of my requirements, still need the other checks from [item-in-site](https://github.com/ArcGIS/opendata-ui/blob/master/packages/ember-arcgis-opendata-components/addon/utils/item-in-site.js#L1) utility.

Reply:
@rweber-esri Moving forward, "associating" something with a site is that it's included in the site's catalog. That's why the `.contains` method was developed. Long-term the item-in-site function would have to be updated to handle the `IHubCatalog` structure, which... given how flexible that is, basically can't be resolved without using something like `.contains` anyhow...